src/View/AdminUser/AddUserUI.java

```diff
@@ -0,0 +1,171 @@
+package View.AdminUser;
+
+import javax.swing.JPanel;
+import javax.swing.JLabel;
+import javax.swing.JTextField;
+import javax.swing.JCheckBox;
+import javax.swing.JComboBox;
+import javax.swing.JButton;
+import javax.swing.SpringLayout;
+import java.awt.BorderLayout;
+import java.awt.Dimension;
+import javax.swing.BoxLayout;
+import javax.swing.BorderFactory;
+import java.util.ResourceBundle;
+
+import Utilities.SpringUtilities;
+import Utils.Constants;
+import View.Events.UserEvent;
+
+/**
+ * UI to Add User, Form with the fields of user
+ *
+ * @author: Lourdes Villca
+ */
+
+public class AddUserUI extends JPanel {
+    private JLabel nameLabel;
+    private JLabel lastNameLabel;
+    private JLabel userLabel;
+    private JLabel passwordLabel;
+    private JLabel activeLabel;
+    private JLabel addressLabel;
+    private JLabel emailLabel;
+    private JLabel roleLabel;
+    private JLabel possitionLabel;
+    private JTextField nameTxt;
+    private JTextField lastNameTxt;
+    private JTextField userTxt;
+    private JTextField passwordTxt;
+    private JCheckBox activeCbox;
+    private JTextField addressTxt;
+    private JTextField emailTxt;
+    private JComboBox roleCbox;
+    private JComboBox possitionCbox;
+    private JButton acceptBtn;
+    private JButton cancelBtn;
+    private UserEvent userEvent;
+    private JPanel leftPanel;
+    private JPanel rightPanel;
+    private ResourceBundle resource;
+
+    /**
+     * Init the components and define a layout for panels
+     */
+    public AddUserUI() {
+
+        resource = resource.getBundle(Constants.APLICATION_RESOURCES);
+
+        leftPanel = buildLeftPanel();
+        rightPanel = buildRightPanel();
+        acceptBtn = new JButton(resource.getString("admin.label.acceptButton"));
```

Comment:
usually the accept button are part of a general group not necessarily admin.label.acceptButton